src/UWP/ArcGISRuntime.UWP.Viewer/ArcGISRuntime.UWP.Viewer.csproj

```diff
@@ -114,6 +114,12 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Screenshots -->
+    <Content Include="Resources\github-markdown-dark.css">
```

Comment:
Don't forget to update the license text to include github-markdown.css; see WPF for the text